docs-src/docs/kognic-io/working_with_scenes_and_inputs.md

```diff
@@ -110,6 +110,34 @@ This would trigger the scene creation process, and once the scene is created, in
 the given batch. If the `batch` parameter is omitted, the latest open batch for the project will be used. We also provide
 a wrapper function `create_inputs` to help with this process, see [Creating Multiple Inputs With One Call](#creating-multiple-inputs-with-one-call).
 
+## List Scenes
+
+:::note
+This feature is new in version 1.6.0
```

Comment:
🤔 This makes me wonder if we should do what a lot of libraries do, and have versioned docs where you need to pick from a dropdown or something, cos we are changing a lot of stuff in the past ~year and will have a 2.0.0

Reply:
I have looked into this and it wasn't straight forward to implement. If we want versioned docs I think that we should have autogenerated docs (with sphinx or similar) for the code. However, the part where we explain things in more detail can remain as it is